app/assets/stylesheets/avalon.css.scss

```diff
@@ -39,12 +39,7 @@
   }
 }
 
-a:not(.btn) {
-  color: $linkColor;
-  &:hover { color: $linkColor; }
-  &:active { color: $linkColor; }
-  &:visited { color: $linkColor; }
-}
+
```

Comment:
This stylesheet was not created by me and has loads of extra spaces. I suppose I shouldn't have added to the mess, but it didn't seem to matter. I'll clean up the entire file.